concourse/tasks/trigger-bigquery-export.yml

```diff
@@ -0,0 +1,20 @@
+platform: linux
+image_resource:
+  type: docker-image
+  source:
+    repository: governmentpaas/cf-cli
+    tag: e1ffec0d1940706f157a8c1e0ab8131b7084fa1c
+inputs:
+  - name: git-main
+    path: src
+
+run:
+  path: sh
+  args:
+    - "-c"
+    - |
+      echo "triggering account-manager export"
+      curl -XPOST -H "Authorization: Bearer ${BEARER_TOKEN}" -H 'Accept: application/json' "https://www.${CDN_DOMAIN}/api/v1/report/bigquery"
```

Comment:
This is interesting `-XPOST`, what's going on here/
I couldn't find any reference to that in the CURL docs, thougt [i could find `](https://curl.se/docs/manpage.html#-X)-X --request` is that what's going on here?

Wasn't sure if there's a missing space or not?

Reply:
It's not missing a space, `-XPOST` and `-X POST` both work

Reply:
Ah, good to know. thanks!